schemas/creativeWork.schema.tpl.json

```diff
@@ -0,0 +1,141 @@
+{
+  "properties": {
+    "about": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add links to the datasets, models or software contained within or described by the creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DatasetVersion",
+        "https://openminds.ebrains.eu/core/ModelVersion",
+        "https://openminds.ebrains.eu/core/SoftwareVersion"
+      ]
+    },
+    "abstract": {
+      "type": "string",
+      "_instruction": "Enter the abstract of the creative work."
+    },
+    "custodian": {
```

Comment:
should this not be "corresponding author" for creative works?

Reply:
I was trying to reuse existing openMINDS terms, I think "corresponding author" has effectively the same role for a journal article as "custodian" for a dataset.

I'm happy to change it, if you think "custodian" is confusing.